include/wlr/xwayland.h

```diff
@@ -322,11 +322,11 @@ void wlr_xwayland_surface_ping(struct wlr_xwayland_surface *surface);
  * Returns: true if the window should receive focus
  *          false if it should be ignored
  */
-bool wlr_xwayland_or_surface_wants_focus(
+bool wlr_xwayland_surface_or_wants_focus(
```

Comment:
While at it we might want to rename to `wlr_xwayland_surface_o_r_wants_focus` or even `wlr_xwayland_surface_override_redirect_wants_focus` to avoid the confusion with the English word "or".

Reply:
Yeah we should really just be explicit here and spell out "override redirect." I actually already did so in zig-wlroots...

Reply:
Also the whole function is just an ugly hack, ideally we should get rid of it and rely on the input model only, but I'm too scared to touch X11 input stuff again.